src/status_im/keycard/core.cljs

```diff
@@ -48,6 +49,16 @@
             (common/listen-to-hardware-back-button)
             (navigation/navigate-replace :keycard-recovery-pin nil)))
 
+(fx/defn load-recovery-pin-screen-with-pairings
+  [{:keys [db] :as cofx}]
+  (let [instance-uid (get-in db [:keycard :application-info :instance-uid])
+        {:keys [pairing paired-on]} ((get-in db [:keycard :pairings]) (keyword instance-uid))]
```

Comment:
```suggestion
        {:keys [pairing paired-on]} (get-in db [:keycard :pairings (keyword instance-uid)])]
```

Reply:
Thanks for the review! All remarks have now been fixed